test/test_backend/routes/tst_graph.py

```diff
@@ -45,57 +45,59 @@
 TEST_CASES = TEST_CASES_FAST
 EXTRA_CONCEPT_IDS = []
 HIDE_VOCABS = ['RxNorm Extension']
-CONNECTED_SUBGRAPH_FILENAME_PATTERN = 'get_connected_subgraph__{}.json'
 
 
 # class TestGraph(unittest.TestCase):
 class TstGraph:
     """Tests for graph.py"""
     test_cases: List[Dict] = []
 
-    # noinspection PyPep8Naming
-    @classmethod
-    def setUpClass(cls):
+    @staticmethod
+    def _get_test_cases() -> List[Dict]:
         """Set up"""
         csv_file = StringIO(TEST_CASES_FAST.strip())
         cases: List[Dict] = list(csv.DictReader(csv_file, delimiter=',', quotechar='"'))
         for case in cases:
             for k, v in case.items():
                 if k == 'codeset_ids':
                     case[k] = v.split(',')
-        cls.test_cases = cases
+        return cases
+
+    # TODO: type hints
+    @staticmethod
+    def _save_output(
+        test_name: str, sg, nodes_in_graph, preferred_concept_ids, orphans_not_in_graph, hidden, minify_threshold=1000
+    ):
+        """Save outputs"""
+        input_path = STATIC_DIR_get_connected_subgraph / f'{test_name}.json'
+        normalized_hidden = {k: sorted(list(v)) for k, v in hidden.items()}
+        results = {
+            'sg.edges': sorted(list(sg.edges)),
+            'nodes_in_graph': sorted(list(nodes_in_graph)),
+            'preferred_concept_ids': sorted(list(preferred_concept_ids)),
+            'orphans_not_in_graph': sorted(list(orphans_not_in_graph)),
+            'hidden': normalized_hidden,
+        }
+        n_records = sum([len(v) for k, v in results.items()])
+        indent = 0 if n_records > minify_threshold else 0
+        with open(input_path, 'w') as f:
```

Comment:
## Tests failing @Sigfried 
@Sigfried These tests are now failing, since the outputs of 1get_connected_subgraph()` have changed since I initially created. Leaving this task up to you now:

- [ ] Do one of the following
  - a. If this is indicative of something being wrong with your recent changes (e.g. when I made the outputs, `get_connected_subgraph()` was correct, but it's not now) --> Fix things and rerun tests until they pass
  - b. if there is no bug in `get_connected_subgraph()` and it is currently outputting what is expected, you need to save new outputs. To do so, run `TstGraph. _create_inputs__get_connected_subgraph()`.